pkg/providers/amifamily/bootstrap/bottlerocket.go

```diff
@@ -74,6 +74,9 @@ func (b Bottlerocket) Script() (string, error) {
 		if b.KubeletConfig.ImageGCLowThresholdPercent != nil {
```

Comment:
@domgoodwin Do you mind valiadating that this pass-through works in functional testing and adding the setting pass-through in the E2E testing as well.

Here's some pointers to each:
1. Functional Testing: https://github.com/aws/karpenter/blob/main/pkg/providers/launchtemplate/suite_test.go#L1135
2. E2E Testing: https://github.com/aws/karpenter/blob/main/test/suites/integration/kubelet_config_test.go#L51